dotCMS/src/main/java/com/dotcms/aspects/interceptors/LocalTransactionalMethodInterceptor.java

```diff
@@ -0,0 +1,26 @@
+package com.dotcms.aspects.interceptors;
+
+import com.dotcms.aspects.DelegateMethodInvocation;
+import com.dotcms.aspects.MethodInterceptor;
+import com.dotcms.business.LocalTransactional;
+import com.dotmarketing.db.LocalTransaction;
+
+/**
+ * Method handler for the {@link LocalTransactional} annotation aspect
+ * @author jsanca
+ */
+public class LocalTransactionalMethodInterceptor implements MethodInterceptor<Object> {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a "static final long serialVersionUID" field to this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2057)